src/components/navigation/Navigation.js

```diff
@@ -58,23 +58,27 @@ class Navigation extends Component {
     }
 
     render() {
-
         const { menuOpen } = this.state;
 
+        let showNavLinks = this.showNavLinks
```

Comment:
seems like this logic should be in `showNavLinks` getter? Also why do we need different logic specifically for `/create-implicit`?

Reply:
if the user already has an account we don't want to show it, unless we do?